ospfd/ospf_te.c

```diff
@@ -2412,6 +2411,7 @@ DEFUN (no_ospf_mpls_te_inter_as,
 
 	/* Deregister the Callbacks for Inter-AS suport */
 	ospf_mpls_te_unregister();
+	OspfMplsTE.inter_as = Off;
```

Comment:
I think this makes sense. But maybe we could move these three lines into the `if` block above, to avoid calling `ospf_mpls_te_unregister()` when the `mpls-te inter-as` command is not previously configured.

Reply:
Renato

You are right. Dimitrii, can you update your patch ?